protocol/synthetix/subgraph/tests/core.test.ts

```diff
@@ -634,6 +634,9 @@ describe('core tests', () => {
   test('handleRewardsDistributed', () => {
     // Needs to be here because of Closures
     const now = new Date(1668448739566).getTime();
+    const newPoolEvent = createPoolCreatedEvent(1, address, now, now - 1000);
+    handlePoolCreated(newPoolEvent);
```

Comment:
Do you want to add some asserts to ensure the pool is assigned correctly to the rewards distributor too

Reply:
Yeah will do